templates/surveys.html

```diff
@@ -4,7 +4,8 @@
   var user_data={% raw user_info_json %};
   var routesObject = {
```

Comment:
lots of `var routesObject = {}` in multiple templates files. This is feeling a little not DRY. Not impeding this pr, but lets think about how this needs to be cleaned up.